### ISSUE TITLE
MySQL backend not working for offline players

### ISSUE THREAD

**Issue by MegaNarwhal** [sgtcaze/NametagEdit#38]
I am attempting to run NTE build 7 from Jenkins with the MySQL backend on 3 servers that are behind a BungeeCord proxy. After some testing, this is what I have found:
When I set my prefix ingame, the change is reflected in the database immediately and the color persists through reconnects.
When the player whose prefix is being changed is offline when the command is executed, their prefix in the ```players``` table does not change.
Joining after setting my prefix in console does show the new prefix ingame, however it is lost when the server restarts even though the plugin finds the entries on startup:
```
[21:40:29 INFO]: [NametagEdit] [MySQL] Found 0 groups
[21:40:29 INFO]: [Vault] Checking for Updates ...
[21:40:29 INFO]: [NametagEdit] [MySQL] Found 1 players
```
When a prefix is changed, the name colors do not propagate to the other servers connected to the database.

**Comment by MegaNarwhal** [comment]
Confirmed to also happen without BungeeCord.

**Comment by sgtcaze** [comment]
This has been addressed for v3.0.1. Thank you.
